README.md

```diff
@@ -15,4 +17,13 @@ Python data structures.
 $ pip install cbor-decoder
 ```
 
+## Run tests
+
+```
+$ pip install -U pytest
```

Comment:
I think we should be using `pip3` here. At least that what succeeded for me installing.